modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/AbstractPlannerTest.java

```diff
@@ -231,6 +232,40 @@ protected static void relTreeVisit(RelNode n, TestVisitor v) {
         return node -> cls.isInstance(node) && pred.test(node);
     }
 
+    protected RelOptPlanner planner(String sql, IgniteSchema publicSchema, String... disabledRules) {
```

Comment:
Why this method isn't used by the method 'physicalPlan'?

Reply:
thanks ! fixed.